package.json

```diff
@@ -47,9 +48,11 @@
   },
   "bin": "cli.js",
   "scripts": {
-    "lint": "make lint",
-    "test": "make lint flow test",
-    "prepublish": "make NIS= dist"
+    "lint": "eslint katex.js server.js cli.js src test contrib dockers",
+    "flow": "flow",
+    "jest": "jest",
+    "test": "check-dependencies && npm run lint && npm run flow && npm run jest",
+    "prepublishOnly": "make NIS= dist"
```

Comment:
`prepublish` is apparently deprecated in npm@5.  Also, running `npm install` was triggering `prepublish` to run.  Hopefully this will still work with our release script.